sources/packages/backend/libs/integrations/src/esdc-integration/student-loan-balances/student-loan-balances.processing.service.ts

```diff
@@ -112,6 +122,15 @@ export class StudentLoanBalancesProcessingService {
             creator: auditUser,
           });
         }
+        if (previousBalanceDate) {
+          // If no previous balance date is present no records exists in the
```

Comment:
can we please have this comment above the if?

Reply:
It will be removed as part of a refactor.

Reply:
It is moved now, please take a look. I did not see how moving outside would make it clear. Please let me know if we need further discussion don't this 😉